Descent3/localization.cpp

```diff
@@ -218,69 +206,39 @@ int LoadStringTables(void) {
 }
 
 // Deallocates all the memory used for the string tables
-void FreeStringTables(void) {
-  DestroyStringTable(String_table, String_table_size);
-  String_table = NULL;
+void FreeStringTables() {
+  String_table.clear();
 }
 
 const char *GetStringFromTable(int index) {
   if ((index < 0) || (index >= String_table_size))
```

Comment:
Here String_table_size better be equal to String_table.size() since we check the index used to access String_table.